motr/idx_dix.c

```diff
@@ -1239,6 +1280,10 @@ static int dix_index_create(struct m0_op_idx *oi)
 
 	M0_ASSERT(dix_iname_args_are_valid(oi));
 
+	if (ENABLE_DTM0)
```

Comment:
Add a helper function for setting oi_flags and add a todo that they needs to be configurable by the user like S3  and add a todo to remove this logic from motr
once configuration option is available in s3.

Reply:
Done.